ably/realtime_presence_integration_test.go

```diff
@@ -254,47 +254,6 @@ func SkipTestRealtimePresence_ServerSynthesized_Leave(t *testing.T) {
 	assert.Empty(t, members)
 }
 
-func TestRealtimePresence_ServerSynthesized_Leave(t *testing.T) {
```

Comment:
This test is removed because unskipped test already covers all assertions with complete details